File: Dddml.Wms.HttpServices.JavaClient/src/org/dddml/wms/domain/SkuId.java
package org.dddml.wms.domain;


public class SkuId
{
    private String productId;

    public String getProductId()
    {
        return this.productId;
    }

    public void setProductId(String productId)
    {
        this.productId = productId;
    }

    private String attributeSetInstanceId;

    public String getAttributeSetInstanceId()
    {
        return this.attributeSetInstanceId;
    }

    public void setAttributeSetInstanceId(String attributeSetInstanceId)
    {
        this.attributeSetInstanceId = attributeSetInstanceId;
    }

    SkuId ()
    {
    }

    public SkuId (String productId, String attributeSetInstanceId)
    {
        this.productId = productId;
        this.attributeSetInstanceId = attributeSetInstanceId;
    }

}

File: Dddml.Wms.AdminUI/ts/SkuId.ts
namespace Dddml.Wms.Domain {

	export class SkuId {

		private productId: string;

		getProductId(): string {
			return this.productId;
		}

		setProductId(productId: string) {
			this.productId = productId;
		}

		private attributeSetInstanceId: string;

		getAttributeSetInstanceId(): string {
			return this.attributeSetInstanceId;
		}

		setAttributeSetInstanceId(attributeSetInstanceId: string) {
			this.attributeSetInstanceId = attributeSetInstanceId;
		}

	}
}

